docs/articles/faq/README.md

```diff
@@ -232,9 +232,15 @@ npm install -g {pluginName}
 
 ### My test fails because TestCafe could not find the required webpage element. Why does this happen?
 
-First, try debugging the tested page with the TestCafe's built-in debugger by adding
-the [t.debug()](../documentation/test-api/debugging.md) method to test code.
-Then run the test and wait until the browser stops at the breakpoint.
+This happens because one of the [selectors](../documentation/test-api/selecting-page-elements/selectors/README.md)
```

Comment:
Better to give link:
... one of the [target elements](https://devexpress.github.io/testcafe/documentation/test-api/actions/#selecting-target-elements) ...
To cover all scenarios